src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
+
+@import "./blocks/page.scss";
+@import "./blocks/calendar.scss";
```

Comment:
add all styles to your commit
<img width="259" alt="image" src="https://user-images.githubusercontent.com/99331793/203019434-d5d5aafa-766d-4f16-8cd8-7c6e03456c82.png">
<img width="298" alt="image" src="https://user-images.githubusercontent.com/99331793/203019450-0a34cdd3-330f-4c51-86bd-a0d087412c69.png">